mmv1/products/networkservices/GrpcRoute.yaml

```diff
@@ -149,6 +149,7 @@ properties:
                         - :EXACT
                         - :REGULAR_EXPRESSION
                       default_value: :EXACT
+                diff_suppress_func: 'tpgresource.ListStringsSuppress([]string{"TYPE_UNSPECIFIED", "EXACT"})'
```

Comment:
Do you happen to know what it means/ what the expected behavior is if users set "TYPE_UNSPECIFIED" or "" explicitly for this field?

Reply:
Then is suppressing the diff of "" and "EXACT" the right way moving forward? Do we see any issue if users set "" and "EXACT" got returned from the API or the other way around?